helm-charts/patches/prow-control-plane/0001-EKS-D-changes-to-helm-control-plane-chart.patch

```diff
@@ -115,29 +125,32 @@ index 4405f98e78..ecb49e4f79 100644
 -          mountPath: /etc/slack
 +        - name: s3-credentials
 +          mountPath: /etc/s3-credentials
-+          readOnly: true
+           readOnly: true
+-        # AWS IAM token needed to assume role to access the prow-build-cluster EKS cluster.
+-        - name: aws-iam-token
+-          mountPath: /var/run/secrets/aws-iam-token/serviceaccount
 +        - name: kubeconfig
 +          mountPath: /etc/kubeconfig
            readOnly: true
 +        - name: shared-bins
 +          mountPath: /shared-bins
 +      initContainers:
-+      - name: aws-iam-authenticator
-+        env:
-+        - name: AWS_STS_REGIONAL_ENDPOINTS
-+          value: regional
-+        image: {{ .Values.awsIamAuthenticator.image }}
-+        command:
-+        - cp
-+        - /aws-iam-authenticator
-+        - /shared-bins/aws-iam-authenticator
-+        volumeMounts:
-+        - name: shared-bins
-+          mountPath: /shared-bins
++        - name: aws-iam-authenticator
```

Comment:
nit: Let's revert these whitespace changes too, to be consistent with YAML indentation across the repo.